From 50c02ef7bc299aa867cfc57623941768e95b8b87 Mon Sep 17 00:00:00 2001
From: Schrijvers Luc <begasus@gmail.com>
Date: Sat, 18 May 2024 09:30:32 +0200
Subject: Use software rendering

Text menus don't display too well all the time with Kirigami
Use fallback to software rendering to solve the display

diff --git a/src/main.cpp b/src/main.cpp
index 04ebf14..f2112ae 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -31,6 +31,9 @@ using namespace Qt::Literals::StringLiterals;
 
 int main(int argc, char *argv[])
 {
+#if defined(Q_OS_HAIKU)
+       qputenv("QT_QUICK_BACKEND", "software");
+#endif
 #if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
     QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
 #endif
-- 
2.45.1

